### ISSUE TITLE
Teams listMembersInOrg does not seem to work with latest Octokit 17.0.0 version.

### ISSUE THREAD

**Issue by kalyani9** [octokit/octokit.js#1635]
Issue Desc: 
Trying to get the list of members (listMembersInOrg)  from Org from latest verison of Octokit. 
Link:
https://octokit.github.io/rest.js/v17/ 
![image](https://user-images.githubusercontent.com/19297798/76026800-7a4fac00-5efd-11ea-8891-c8588d02bcee.png)

But I am getting HTTP NOT FOUND 404. Are there any compatibility issues with latest version of Octokit ?  
octokit.teams.list works as expected. 

Please advise

**Comment by gr2m** [comment]
There are no compatibility issue that I'm aware of. Can you please create a minimal, reproducible test case?

**Comment by Jarlotee** [comment]
@kalyani9  are you using github enterprise?

**Comment by howdiz** [comment]
I'm having this issue as well @kalyani9.  Yes, I'm using GitHub Enterprise, @Jarlotee.

here is my test case @gr2m : 

`octokit.teams.getByName({
    org: 'Linode',
    team_slug: 'Developers',
  })
  .then(({ data }) => {
    console.log(data);
});`
works 

however,
`octokit.teams.listMembersInOrg({
    org: 'Linode',
    team_slug: 'Developers',
  })
  .then(({ data }) => {
    console.log(data);
});`

returns: 
`HttpError: Not Found`

**Comment by kalyani9** [comment]
Yes @howdiz , I happened to face issue with version 17.00. However, I had a work around with octokit version 16.43.1  by using octokit.teams.listMembers . This seems to be deprecated but works fine.

**Comment by gr2m** [comment]
The endpoint behind `octokit.teams.listMembersInOrg()` ([`GET /orgs/:org/teams/:team_slug/members`](https://developer.github.com/v3/teams/members/#list-team-members)) was only recently introduced, it does not exist on GitHub Enterprise Server yet. So if you use GitHub Enterprise Server, that is the reason. 

To solve this, I would recommend not to use `@octokit/rest`, but instead build your own, by importing the endpoint methods for your GHES version from https://github.com/octokit/plugin-enterprise-server.js. The code would look like this:

```js
const { Octokit } = require("@octokit/core")
const {  enterpriseServer220 } = require("@octokit/plugin-enterprise-server")
const { requestLog } = require("@octokit/plugin-request-log");
const { paginateRest } = require("@octokit/plugin-paginate-rest");

const MyGheOctokit = Octokit.plugin(enterpriseServer220, requestLog, paginateRest).defaults({
  userAgent: `my-company-octokit/1.0.0`,
});

const octokit = new MyGheOctokit({
  baseUrl: "https://[hostname]/api/v3" // your GHES API base URL,
  // any other options such as `auth`
})
```

Now `octokit` has only the methods for the endpoints that are available to GHES v2.20, including the enterprise admin endpoints.

---

If you don't use GitHub Enterprise Server, then the problem might be an OAuth Scope issues. Make sure the ` admin:org` scope is enabled. You can check the scopes for the current token with this code:

```js
const { headers } = await octokit.request("HEAD /")
console.log(headers["x-oauth-scopes"]);
```
